_pages/webextensions.md

```diff
@@ -12,160 +12,173 @@ parent: activities
 
 
 ## Planning Your Workshop
 
-Help contribute to the open web by running an extension development workshop. This activity can also be run solo.
+Read the [event guide](https://activate.mozilla.community/eventguide/) on how to set up an event page and how to organize an event. For this campaign, we recommend scheduling a series of 3 workshops and dedicating 1.5 - 4 hours for each workshop. Be sure to complete the actions listed in the Pre-Event section 2 - 3 days before hosting the event. 
 
-1. Read the [event guide](/eventguide/) on how to set up an event page and how to organize an event.
-2. Choose whether this will be self-facilitated or co-facilitated with someone who has experience developing browser extensions
-3. Get the word out! Invite your community to the event. If this activity is run together with a Campus Club, they can customize a flyer with your event information for print or electronic distribution.
-4. 2 - 3 days before the event, ask participants to complete the Pre-Event section on their own.
+Attendees should bring their own laptops to the workshop, or the workshop should be held in a computer lab.
+
+Interested in learning more about extension development or sharing additional information with workshop attendees? See the Resources section below. 
+
+
+## Flow of Workshops 
+
+### Pre-Event
+
+To ensure that your participants get to focus on building extensions during the workshop
+
+* Ask registered participants to explore addons.mozilla.org, install a few extensions, and see how they work. 
+
+* Make sure that computers being used for the workshop have the most recent version of Firefox Developer Edition (available for Windows, Linux, or Mac OS X). If attendees are bringing their own computers, ask them to install Firefox Developer Edition. 
+
+
+
+### First Meeting: Introduction to Browser Extensions 
+
+The focus of this meeting will be to introduce a general audience to browser extensions and the WebExtensions API. You can make a copy of the presentation [Introduction to Browser Extensions](https://docs.google.com/presentation/d/1SwKbcfI55DEgjQMAzNHvDY5e4ywCUgsWu-qtt2LsxeE/edit#slide=id.p) and customize it for your group. 
+
+We recommend the following agenda for this event: 
+
+* Introduce participants to browser extensions and why they are important.  Here is some sample language you can use: 
+---
+    *“Welcome to the [name of event]! I am [name]. Today, we are going to customize our Firefox browsers by developing our own extensions to solve real problems in our own lives. Add-ons are a great way to add extra features to your browser, like blocking time-wasting websites during study hours, or turning images into cat pictures. With WebExtensions APIs, it’s now easier to create add-ons that are compatible with Firefox, Chrome, and Opera. Add-ons help keep users to be in control of their online experience, so your support in building them and the community around them matter.”*
```

Comment:
the first list item still gets rendered too big, this is because of the following "---". Can you remove the "---" above and below the example text? Then it will show correctly:

![screen shot 2018-03-27 at 23 25 16](https://user-images.githubusercontent.com/330324/37995875-22c5a388-3216-11e8-87e6-03bbd1797647.png)